test/README.md

```diff
@@ -0,0 +1,209 @@
+# Automation test framework for Web Chat by using Direct Line API
+
+> ## Disclaimer : The target audience for this document is towards to Web Chat and Direct Line API contributors. Although Direct Line API is underneath the Bot Framework, but to test bots locally, please use the [Microsoft Bot Emulator](https://github.com/Microsoft/BotFramework-Emulator) instead.
+
+## Overview of the test framework eco-system
```

Comment:
I realized a couple more spelling errors. I corrected them on the newest commit.